### DISCUSSION TITLE
too many mining resources?

### DISCUSSION THREAD

**Discussion by NoTitleGamesOfficial** [azerothcore/azerothcore-wotlk#18240] [General]
I'm using azerothcore for the first time and I noticed that there are way to much mining resources. Sometimes I discover places where there like 10 resources within the minimap - much more than 'blizzlike' should have. 
Is this a settings thing?

**Comment by NoTitleGamesOfficial** [comment]
Something like this:
https://mega.nz/file/tSJh1Y7J#20gzT0L3C_WoNIs60Fae0LY8ppiBxvNeAEOYSWaMA6s

**Comment by Rylkiden** [comment]
I can confirm this as well. I notice it most in Loch Modan. Whereas Dun Morogh it feels like I have to really search out nodes even with addons, Loch Modan has essentially every mining spot respawning frequently with overlapping tin and silver deposits. What's interesting there is the copper seems to behave correctly (having to search for the deposits across the map), it's only the tin and silver that are behaving weirdly. 

From what I have been able to find, it is an issue with the pools, based on what I've found in topics like #2211. 

If anyone can point me to some guides on how to compare the nodes to the database, I'll fiddle around to see if I can bring those more into line

**Reply by NoTitleGamesOfficial** [reply]
I can confirm that on Bloodmyst Isle its also tin and silver deposits that respawn very quick. 
Sometimes they respawn within 1-2 minutes.
